staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -1883,6 +1883,12 @@ type MigrationConfiguration struct {
 	DisableTLS                        *bool              `json:"disableTLS,omitempty"`
 }
 
+// DiskVerification holds container disks verification limits
+// +k8s:openapi-gen=true
+type DiskVerification struct {
+	MemoryLimitBytes *resource.Quantity `json:"memoryLimitBytes"`
```

Comment:
The `Bytes` is normally not added in k8s APIs. `MemoryLimit` should be enough.

Reply:
Maybe add a description string which says something like:

```
Maximum amount of memory which can be used by the disk verification process to avoid Attacks via malformed disk images.
```

Reply:
Gotcha, fixed!